### DISCUSSION TITLE
How would I enable the PCI Bus Master Flag for a MMIO (Memory Mapped I/O) device?

### DISCUSSION THREAD

**Discussion by elkaamee326** [CosmosOS/Cosmos#2071] [Q&A]
Also, how can I know if the master bus is enabled? Thanks.

**Comment by MishaTy** [comment]
You can use the EnableDevice() function

**Reply by elkaamee326** [reply]
Isn't there a function called 
```CSharp 
EnableMemory(bool Enable) { ... }
```
what does it do?

**Reply by MishaTy** [reply]
It enables memory I/O on the device

**Reply by elkaamee326** [reply]
Oh, so it doesn't enable the Master Bus?

**Reply by MishaTy** [reply]
Yeah

**Reply by elkaamee326** [reply]
Thanks :)
